src/main/kotlin/ru/asipes/doppler/controller/ConfigurableServicesController.kt

```diff
@@ -0,0 +1,29 @@
+package ru.asipes.doppler.controller
+
+import org.springframework.stereotype.Component
+import org.springframework.web.servlet.ModelAndView
+import org.springframework.web.servlet.mvc.AbstractController
+import ru.asipes.doppler.response.AbstractResponse
+import ru.asipes.doppler.response.ResponsesHolder
+import java.util.stream.Collectors
+import javax.servlet.ServletResponse
+import javax.servlet.http.HttpServletRequest
+import javax.servlet.http.HttpServletResponse
+
+@Component
+class ConfigurableServicesController(private val responsesHolder: ResponsesHolder): AbstractController() {
+    @Throws(Exception::class)
+
+    override fun handleRequestInternal(request: HttpServletRequest, response: HttpServletResponse): ModelAndView? {
+        val endpointResponse: AbstractResponse = responsesHolder[request.requestURI]
+        response.status = endpointResponse.getStatus()
```

Comment:
```suggestion
        response.apply {
            status = endpointResponse.getStatus()
            characterEncoding = endpointResponse.getCharacterEncoding()
            body = endpointResponse.getResponse(request)
        }
```
Можно юзать scope-функции